tutorials/ingest-real-time-websocket-data.md

```diff
@@ -90,36 +103,41 @@ websocket object and establish connection.
 
 *   `symbols`
 
-    This argument needs to be a list of stock ticker symbols (for example, `MSFT`) or
-    crypto trading pairs (for example, `BTC/USD`). When using a websocket connection you
-    always need to subscribe to the events you want to receive. You can do this
-    by using the `symbols` argument or if your connection is already
-    created you can also use the `subscribe()` function to get data for additional
-    symbols.
+    This argument needs to be a list of stock ticker symbols (for example,
+    `MSFT`) or crypto trading pairs (for example, `BTC/USD`). When using a
+    websocket connection you always need to subscribe to the events you want to
+    receive. You can do this by using the `symbols` argument or if your
+    connection is already created you can also use the `subscribe()` function to
+    get data for additional symbols.
 
 <Procedure>
 
 ### Connecting to the websocket server
 
 1.  Create a new Python file called `websocket_test.py` and connect to the
-    Twelve Data servers using the wrapper library:
+    Twelve Data servers using the `<YOUR_API_KEY>`:
 
     ```python
-    # websocket_test.py:
-    from twelvedata import TDClient
-    
-    def on_event(event):
-     print(event) # prints out the data record (dictionary)
-    
-    td = TDClient(apikey="TWELVE_DATA_APIKEY")
-    ws = td.websocket(symbols=["BTC/USD", "ETH/USD"], on_event=on_event)
-    ws.connect()
-    ws.keep_alive()
+       import time
+       from twelvedata import TDClient
+
+        messages_history = []
+
+        def on_event(event):
+         print(event) # prints out the data record (dictionary)
+         messages_history.append(event)
+
+       td = TDClient(apikey="<YOUR_API_KEY>")
+       ws = td.websocket(symbols=["BTC/USD", "ETH/USD"], on_event=on_event)
+       ws.subscribe(['ETH/BTC', 'AAPL'])
+       ws.connect()
+       while True:
+       print('messages received: ', len(messages_history))
+       ws.heartbeat()
+       time.sleep(10)
     ```
 
-    Make sure to pass your API key as an argument for the `TDClient` object.
-
-1.  Now run the Python script:
+1.  Run the Python script:
 
     ```bash
     python websocket_test.py
```

Comment:
I can't suggest it, L146 is a result statement, so it should be in the same step as the previous. I'm also a bit 🫤 on the "immediately", maybe say "When you run the script, you receive a response from the server ..." ? 

Also, it's not highlighting properly. Could be a whitespace problem?

![image](https://github.com/timescale/docs/assets/3914967/ef8c63b8-6f9c-4883-994f-7df94d4750b4)